src/wind_energy/ABL.cpp

```diff
@@ -86,11 +86,26 @@ void ABL::pre_advance_work()
     {
         // TODO: This should be handled by PlaneAveraging
         int output_interval = 1;
+        int plot_type = 0;
         amrex::ParmParse pp("io");
         pp.query("line_plot_int", output_interval);
+        pp.query("line_plot_type", plot_type);
 
         if ((output_interval > 0) && (time.time_index() % output_interval == 0)) {
-            m_pa.plot_line_text("line_plot.txt", time.time_index(), time.current_time());
+            switch (plot_type) {
```

Comment:
Why not consolidate this logic in `PlaneAveraging` and just pass `plot_type` to the `m_pa.process_outputs()` method?

Reply:
good idea, I can do that